src/extension.ts

```diff
@@ -6,20 +6,26 @@ import { StatusBarItem } from './statusBarItem'
 import { FlatpakRunner } from './flatpakRunner'
 import { TaskMode } from './taskMode'
 import { restoreRustAnalyzerConfigOverrides } from './integration/rustAnalyzer'
-import { findManifests } from './flatpakManifestUtils'
+import { findManifests, versionCompare } from './flatpakManifestUtils'
 import { FlatpakTerminal } from './flatpakTerminal'
+import { execSync } from 'child_process'
+import { Command } from './command'
 const { executeCommand, registerCommand } = commands
 
 export const EXT_ID = 'flatpak-vscode'
 
 // whether VSCode is installed in a sandbox
 export const IS_SANDBOXED = existsSync('/.flatpak-info')
+// Currently installed Flatpak version
+export const FLATPAK_VERSION = execSync((new Command('flatpak', ['--version'])).toString()).
```

Comment:
Compiled version doesn't really like the usage of `new Command` here...

Reply:
very weird indeed

Reply:
https://stackoverflow.com/a/34582568/14745745 might be related

Reply:
There is nothing that ensures the Command module is loaded before which makes this part break and i don't know if there is a way to force loading another module first

Reply:
we're probably making circular dependency with Command? Maybe put IS_SANDBOXED inside the command module

Reply:
Switched back to whatever we used to do, didn't want to go with a duplicated logic of what Command does internally